Split the installation section & add the bootstrap sub section (#1568)
fluxcd/website: content/en/flux/installation/bootstrap/aws-code-commit.md

---
title: AWS Code Commit
linkTitle: AWS Code Commit

> description: "How to follow bootstrap procedure for Flux using AWS Code Commit"

Reviewer: ```suggestion
description: "How to bootstrap Flux with AWS Code Commit"
```

Please change all the others pages to this shorter format.
